molgenis/molgenis-emx2
# fix(directory): biobanks juridical_person is required

What are the main changes you did:
- adjusted biobank-directory molgenis.csv and biobank-directory/stagingArea molgenis.csv => put required from false to true for Biobanks juridical_person
- it turned out that stagingArea's demo data files contained ERIC-only columns (f.e. national_node), these columns are removed from the files.

how to test:
- create schema's with both the biobank_directory template (including demo-data) and the biobank_directory_staging template (including demo-data) and see in f.e. forms that Biobank juridical person is required.

todo:
- [nvt ] added/updated tests
- [nvt] added/updated testplan to include a test for this fix, including ref to bug using # notation
